src/aleph_client/commands/instance/display.py

```diff
@@ -291,9 +291,9 @@ async def _prepare_instance_column(self):
             aleph_price = Text(f"{displayable_amount(required_tokens, decimals=3)} (fixed)", style="magenta3")
         else:
             psec = f"{displayable_amount(required_tokens, decimals=8)}/sec"
-            phour = f"{displayable_amount(3600*required_tokens, decimals=3)}/hour"
-            pday = f"{displayable_amount(86400*required_tokens, decimals=3)}/day"
-            pmonth = f"{displayable_amount(2628000*required_tokens, decimals=3)}/month"
+            phour = f"{displayable_amount(3600 * required_tokens, decimals=3)}/hour"
+            pday = f"{displayable_amount(86400 * required_tokens, decimals=3)}/day"
+            pmonth = f"{displayable_amount(2628000 * required_tokens, decimals=3)}/month"
```

Comment:
More formatting-only changes, unclear why this happened? Check that your IDE uses the correct version of `black`.